### PR TITLE
Wip solution

### PR DESCRIPTION
**Pull request by vicfriedman**
@pletcher @AnnJohn

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -10,4 +11,17 @@
     expect(@user.email).to match 'user@example.com'
   end
 
+  it "#role is :user by default" do
+    expect(@user.role).to eq "user"
+  end
+
+  it "#role can be set to :vip" do
+    @user.role = :vip
+    expect(@user.role).to eq "vip"
```
Comment (pletcher) [inline spec/models/user_spec.rb]: This is a really weird Devise-ism, right? It's confusing to me that we assign symbols to the role but get back strings. I guess this is okay, though.

**Comment by pletcher** [comment]
LGTM :shipit:

**Comment by pletcher** [comment]
Can we add .learn, CONTRIBUTING.md and LICENSE.md to the `solution` and `wip-solution` branches? Then this should be good to go.

**Comment by vicfriedman** [comment]
yes, adding now - @SophieDeBenedetto wrote an awesome gem that creates the contributing and licensing and .learn files for you. I forgot the name thoguh...

**Comment by SophieDeBenedetto** [comment]
@vicfriedman @pletcher, gem for automatically writing those files is here https://github.com/SophieDeBenedetto/learn-write

gem install `learn_writer` then run `learn-write` in the directory of whatever repo you want to add those files to.

**Pull request merged**